Add true prerequisite of CS438

YangchenYe323 opened the pull request.

YangchenYe323 committed:
Add true prerequisite of CS438

ziyue5 commented:
Thanks so much for your contribution! May I know the reason we changed the category session structure? Looks like we don't need direct to category session using the in-page anchor.

YangchenYe323 commented:
> Thanks so much for your contribution! May I know the reason we changed the category session structure? Looks like we don't need direct to category session using the in-page anchor.

Updated. Seems the editor auto-completed that section.

ziyue5 reviewed (approved):
Everything is good. Thanks!
